File: leetcode2/1easy/이준열/2566.cpp
class Solution {
    public:
        int minMaxDifference(int num) {
            string conv = to_string(num);
            string minStr = conv;
            string maxStr = conv;
            char minWord = conv[0];
            char maxWord = -1;
            for (int i = 0; i < conv.size(); i++)
            {
                if (conv[i] == minWord)
                {
                    minStr[i] = '0';
                }
                if (maxWord == -1 && conv[i] != '9')
                {
                    maxWord = conv[i];
                    maxStr[i] = '9';
                }
                if (conv[i] == maxWord)
                {
                    maxStr[i] = '9';
                }
            }
            int minVal = stoi(minStr);
            int maxVal = stoi(maxStr);
    
            return maxVal - minVal;
        }
    };

File: leetcode2/1easy/유시연/2566.py
class Leetcode.이재훈.Solution:
    def minMaxDifference(self, num: int) -> int:
        str_num = str(num)
        max_val = num
        min_val = num

        for d in set(str_num):
            if d != '9':
                max_candidate = int(''.join('9' if ch == d else ch for ch in str_num))
                max_val = max(max_val, max_candidate)

        for d in set(str_num):
            if d != '0':
                min_candidate = int(''.join('0' if ch == d else ch for ch in str_num))
                min_val = min(min_val, min_candidate)

        return max_val - min_val
